src/utils.c

```diff
@@ -27,9 +27,12 @@
 
 */
 
+#include <src/libdogecoin-config.h>
```

Comment:
This could be with quotes instead to indicate local inclusion and consistency with how it's being included elsewhere, but it's such a minor thing
`
#include "src/libdogecoin-config.h"
`